src/components/ConsentPicker.vue

```diff
@@ -0,0 +1,486 @@
+            <i :class="param.icon" style="font-size: 1rem; width: 20%"></i>
+          </div>
+        </div>
+        <div class="border-solid border-round mt-3" style="width: 70%; cursor: pointer">
+          <div class="mt-1 mb-1 ml-2 text-center flex cursor-pointer">
+            <div class="flex flex-row w-full cursor-pointer">
+              <PvCheckbox
+                v-model="paramCheckboxData"
+                input-id="default-params"
+                value="hasDefault"
+                @change="checkBoxStatus"
+              />
+              <label class="ml-2 mr-3 flex cursor-pointer" style="width: 80%" for="default-params"
+                >Default Data Collection Values</label
+              >
+              <i class="pi pi-align-justify" style="font-size: 1rem; width: 20%"></i>
+            </div>
+          </div>
+        </div>
+        <h3 class="mt-5">Additional Data Collection</h3>
+        <div class="border-solid border-round" style="width: 70%; cursor: pointer">
+          <div class="mt-1 mb-1 ml-2 text-center flex cursor-pointer">
+            <div class="flex flex-row w-full cursor-pointer">
+              <PvCheckbox v-model="specialParam" input-id="video-recording" value="hasVideo" @change="checkBoxStatus" />
+              <label class="ml-2 mr-3 flex cursor-pointer" style="width: 80%" for="video-recording"
+                >Video Recording</label
+              >
+              <i class="pi pi-video" style="font-size: 1rem; width: 20%"></i>
+            </div>
+          </div>
+        </div>
+        <div class="border-solid mt-2 border-round" style="width: 70%; cursor: pointer">
+          <div class="mt-1 mb-1 ml-2 text-center flex cursor-pointer">
+            <div class="flex flex-row w-full cursor-pointer">
+              <PvCheckbox v-model="specialParam" input-id="audio-recording" value="hasAudio" @change="checkBoxStatus" />
+              <label class="ml-2 mr-3 flex cursor-pointer" style="width: 80%" for="audio-recording"
+                >Audio Recording</label
+              >
+              <i class="pi pi-phone" style="font-size: 1rem; width: 20%"></i>
+            </div>
+          </div>
+        </div>
+        <div class="border-solid mt-2 border-round" style="width: 70%; cursor: pointer">
+          <div class="mt-1 mb-1 ml-2 text-center flex cursor-pointer">
+            <div class="flex flex-row w-full cursor-pointer">
+              <PvCheckbox
+                v-model="specialParam"
+                input-id="eye-tracking"
+                value="hasEyeTracking"
+                @change="checkBoxStatus"
+              />
+              <label class="ml-2 mr-3 flex cursor-pointer" style="width: 80%" for="eye-tracking">Eye - Tracking</label>
+              <i class="pi pi-eye" style="font-size: 1rem; width: 20%"></i>
+            </div>
+          </div>
+        </div>
+        <div class="hidden">
+          <h3 class="mb-4 mt-4">Consent Amount and Expected Time</h3>
```

Comment:
If I'm understanding correctly, should this header be `Compensation Amount` or `Experiment Compensation` instead of `Consent Amount`?